tests/tests_suite_test.go

```diff
@@ -123,6 +123,6 @@ var _ = ReportBeforeEach(func(specReport SpecReport) {
 	k8sReporter.JustBeforeEach(CurrentSpecReport())
 })
 
-var _ = ReportAfterEach(func(specReport SpecReport) {
+var _ = JustAfterEach(func() {
```

Comment:
Please explain in the commit message the motivation for this change (e.g The test clean is done `AfterEach` test, so the report collection must be invoked before the resources of the test are cleaned).